pandas/core/generic.py

```diff
@@ -11382,7 +11382,41 @@ def all(
             name2=name2,
             axis_descr=axis_descr,
             notes="",
-            examples="",
+            examples="""
+
+            Examples
+            --------
+            >>> s = pd.Series([1, 2, 3])
+            >>> s.sem().round(6)
+            0.57735
```

Comment:
just to make it clearer to people what the method's doing, perhaps we could add below something like
```
This should approximately match `np.std([1, 2, 3], ddof=1) / np.sqrt(3)`
```
?